pkg/asset/installconfig/aws/aws.go

```diff
@@ -53,6 +55,24 @@ func Platform() (*aws.Platform, error) {
 	regionTransform := survey.TransformString(func(s string) string {
 		return strings.SplitN(s, " ", 2)[0]
 	})
+
+	defaultRegion := "us-east-1"
+	var defaultRegionName string
```

Comment:
Should we assign this as well, in case `defaultRegionPointer` is nil?

Reply:
> Should we assign this as well, in case `defaultRegionPointer` is nil?

I've pushed ed50028 -> 9dec9df, which takes a slightly different approach, but should address your point.